tools/packaging/generate_changelog.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# This would generate a change log required for build Debian installation package.
+# Don't run this script on your local machine, run this inside docker
+# you would need to install git client as well as moreutils
```

Comment:
how moreutils and debhelper are used here?

Reply:
we need moreutils for sponge command. debhelper is not required for for this script.